gui/gui/src/TGScrollBar.cxx

```diff
@@ -306,7 +306,7 @@ Bool_t TGScrollBarElement::HandleCrossing(Event_t *event)
 /// Constructor.
 
```

Comment:
Could you add a \param doxygen description here? thanks!!

Reply:
Done in [51f5572](https://github.com/root-project/root/pull/16993/commits/51f55724f1886a82da5773adbed0df05916d7d4f). I'm not familiar with the class so it's a "best effort" documentation